### PR TITLE
feat : Added-token-generation-create-room-and-delete-room

### PR DESCRIPTION
**Pull request by SimardeepSingh1450**
Partially resolves Issue : #12 and #13 (sets up create-room and delete-room for livekit-server-sdk)
Fully Resolves : #9  (Livekit-server-sdk setup)
Changes :

1. Added TokenGeneration Function, currently the token generation endpoint takes 2 inputs -> Room Name and Participant Name.
2. Added CreateRoom Function, currently this function takes 1 input -> Room Name.
3. Added DeleteRoom Function, currently this function takes 1 input -> Room Name.
4. Added Controllers and Routes folders for a better folder structure.
5. Added 'livekit-server-sdk' and 'dotenv' as dependency.
6. Setup livekit-server-sdk for managing rooms and token creation.
7. Added instructions in readme on how to setup API keys for livekit-server-sdk with instructions and source.

### CODE REVIEW

**Review by chandansgowda — CHANGES_REQUESTED** [review]
1. Change the file names to follow CamelCase.
2. Add instructions for contributors on how to place thier API keys in env file.

**Comment by chandansgowda** [comment]
Also, the PR just partially resolves the referenced issues since Firebase room data is not being created or deleted. Please change the description accordingly.

**Review by chandansgowda — APPROVED** [review]

**Review by chandansgowda — CHANGES_REQUESTED** [review]
I think it would be better to have CRUD operations related to LiveKit Room in one file rather than having a separate file for each function.

File: controllers/createRoom.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        };
+        svc.createRoom(opts).then((room) => {
+            console.log('room created', room);
+            res.json({"msg":"Room created Successfully","room":room});
+        });   
+    }catch(error){
+        console.log(error);
+        res.json({"msg":"Error creating room"});
+    }
+    }
```
Comment (jddeep) [inline controllers/createRoom.js:30]: please do proper code formatting

**Review by jddeep — CHANGES_REQUESTED** [review]

**Review by chandansgowda — APPROVED** [review]
Looks good to me.

**Review by jddeep — APPROVED** [review]

**Pull request merged**